alarm_table_defs.cpp

```diff
@@ -133,14 +133,25 @@ bool AlarmTableDefs::populate_map(std::string path,
     for (s_it = a_it->_severity_details.begin(); s_it != a_it->_severity_details.end(); s_it++)
     {
       AlarmTableDef def(*a_it, *s_it);
-      AlarmTableDefKey key(a_it->_index, s_it->_severity);
-      _key_to_def.emplace(key, def);
+      AlarmTableDefs::insert_def(def);
     }
   }
-
+  AlarmDef::SeverityDetails cleared(AlarmDef::CLEARED, "Test alarm cleared description", "Test alarm cleared details");
```

Comment:
I think we had discussed only putting this code in `clearwater-fv-test` - as it is, this will add a line saying "Test alarm cleared description" to real servers, which I don't think is what we want. (If you think we do want that, let's discuss.)